bears/general/InvalidLinkBear.py

```diff
@@ -79,7 +81,8 @@ def find_links_in_file(file, timeout, link_ignore_regex):
             match = regex.search(line)
             if match:
                 link = match.group()
-                if not link_ignore_regex.search(link):
+                if not (link_ignore_regex.search(link) or
```

Comment:
the 'or' as well as 'not' both should be there because we want to display info about invalid links which are 'not' present in ```link_ignore_regex``` or 'not' present in ```link_ignore_list``` that is why outer ```not``` and inner ```or``` is present.